automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/methodrepo/MethodRepoSpec.scala

```diff
@@ -21,7 +21,7 @@ class MethodRepoSpec extends FreeSpec with MethodFixtures with UserFixtures with
 
         // create it
         val name = "TEST-CREATE-" + randomUuid
-        val attributes = MethodData.SimpleMethod.creationAttributes + ("name" -> name)
+        val attributes = MethodData.SimpleMethod.creationAttributes + ("name" -> name) + ("documentation" -> "documentation")
```

Comment:
Suggest moving "documentation" string to `MethodData.scala` in https://github.com/broadinstitute/workbench-libs/blob/3fba64145ce7b67e3f3ea08f91aae3e1bbc6e1c6/serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/fixture/MethodData.scala

Reply:
I didn't want to change that data and potentially mess up other tests.

Reply:
That's a legit concern. I looked through the other tests and I don't think it would affect them. So the question is whether it's okay to make that change to methods that are marked as "DO_NOT_CHANGE"..
Paging @ahaessly or @breilly2 for a second opinion on A) can we edit the DONOTCHANGE methods and B) is it worth adding method descriptions to these methods